core/src/impl/Kokkos_ClockTic.hpp

```diff
@@ -90,6 +90,26 @@ KOKKOS_IMPL_HOST_FUNCTION inline uint64_t clock_tic_host() noexcept {
 
   return (uint64_t)cycles;
 
+#elif defined(__ppc__)
+
+  unsigned long long int result = 0;
+  unsigned long int upper, lower, tmp;
```

Comment:
I think naming these types in terms of the explicit bit sizes is somewhat preferable:
```suggestion
  uint64_t result = 0;
  uint32_t upper, lower, tmp;
```
Of course, that's if the relevant compilers and assemblers allow for it.

Reply:
@PhilMiller Fixed.